server/utils/courseParticipationUtils.test.ts

```diff
@@ -354,68 +356,145 @@ describe('CourseParticipationUtils', () => {
-      })
+            it('displays the status on its own', () => {
+              const { rows } = CourseParticipationUtils.summaryListOptions(withoutOutcomeYearStarted, referralId)
+              expect(getRowValueText(rows, 'Outcome')).toEqual('Incomplete')
+            })
+          })
+        })
+
+        describe('when the outcome is complete', () => {
+          describe('and there is a yearCompleted value', () => {
+            const withOutcomeYearCompleted = {
+              ...courseParticipationWithName,
+              outcome: { status: 'complete' as CourseParticipationOutcome['status'], yearCompleted: 2019 },
+            }
+
+            it('displays the outcome and yearCompleted value', () => {
+              const { rows } = CourseParticipationUtils.summaryListOptions(withOutcomeYearCompleted, referralId)
+              expect(getRowValueText(rows, 'Outcome')).toEqual('Complete, Year complete 2019')
+            })
+          })
+
+          describe('and there is no yearCompleted value', () => {
+            const withoutOutcomeYearCompleted = {
+              ...courseParticipationWithName,
+              outcome: { status: 'complete' as CourseParticipationOutcome['status'], yearCompleted: undefined },
+            }
 
-      it('only shows the type in the Setting row when setting location is undefined', () => {
-        const withoutSettingLocation = {
-          ...courseParticipationWithName,
-          setting: { location: undefined, type: 'community' as CourseParticipationSetting['type'] },
-        }
+            it('displays the status on its own', () => {
+              const { rows } = CourseParticipationUtils.summaryListOptions(withoutOutcomeYearCompleted, referralId)
+              expect(getRowValueText(rows, 'Outcome')).toEqual('Complete')
+            })
+          })
+        })
 
-        const { rows } = CourseParticipationUtils.summaryListOptions(withoutSettingLocation, referralId)
-        const settingRow = getRow(rows, 'Setting')
+        describe('when there is no outcome', () => {
+          const withoutOutcome: CourseParticipationWithName = {
+            ...courseParticipationWithName,
+            outcome: {},
+          }
 
-        expect(settingRow).toEqual({ key: { text: 'Setting' }, value: { text: 'Community' } })
+          it('displays "Not known"', () => {
+            const { rows } = CourseParticipationUtils.summaryListOptions(withoutOutcome, referralId)
+            expect(getRowValueText(rows, 'Outcome')).toEqual('Not known')
+          })
+        })
       })
 
-      it('only shows the status in the Outcome row when yearStarted is undefined on an incomplete outcome', () => {
-        const withoutOutcomeYearStarted = {
-          ...courseParticipationWithName,
-          outcome: { status: 'incomplete' as CourseParticipationOutcome['status'], yearStarted: undefined },
-        }
+      describe('additional detail', () => {
```

Comment:
There's no "all information is provided" condition here (or for source below) like with setting and outcome. I guess it's included in the default test, but so are some of the setting and outcome combinations

Reply:
Makes sense